bodhi/client/bindings.py

```diff
@@ -80,6 +80,9 @@ def __unicode__(self):
         """
         return u'Update not found: {}'.format(self.update)
 
+    # Use __unicode__ method under __str__ name for Python 3
+    __str__ = __unicode__
```

Comment:
https://github.com/fedora-infra/bodhi/pull/2153 also adds a ```__str__()``` method here. Would you care to comment over there so the two PRs coordinate a bit?

Reply:
Comment added. I'll solve all conflicts at the end of porting process because if I would rebase frequently, it would be harder to see recent changes in this PR and do a review.